app/validators/sections/section_validator.py

```diff
@@ -40,6 +41,24 @@ def validate_summary(self):
             for item in section_summary.get("items", []):
                 self.validate_list_exists(item.get("for_list"))
 
+    def validate_section_enabled(self):
+        section_enabled = self.section.get("enabled", None)
+
+        if section_enabled and isinstance(section_enabled, list):
+            for enabled in self.section.get("enabled"):
```

Comment:
Worth reusing the already existing var `section_enabled` instead of access again.

Reply:
Using the var now, thanks!